proxy_docker/app/script/requesthandler.sh

```diff
@@ -284,6 +284,13 @@ main()
           response_to_client "${response}" ${?}
           break
           ;;
+        getmempoolinfo)
+          # curl GET http://192.168.111.152:8080/getmempoolinfo
+          trace "trace: $(get_mempool_info)"
+          response=$(get_mempool_info "${line}")
```

Comment:
get_mempool_info is called twice.  Please trace response variable or data variable inside the get_mempool_info function.